.env.test

```diff
@@ -4,6 +4,5 @@ FLASK_ENV=development
 
 # Change the following values for local development.
 SECRET_KEY=secret-key
-TRELLO_API_KEY=
-TRELLO_API_TOKEN=test-key
-TRELLO_BOARD_ID=
\ No newline at end of file
+MONGO_DB_NAME=listcollection
```

Comment:
Add new setup to `readme.md` and remove references to Trello, if you have any across the project